managed/services/user/user.go

```diff
@@ -32,25 +32,34 @@ import (
 type Service struct {
 	db *reform.DB
 	l  *logrus.Entry
+	c  clientInterface
 
 	userpb.UnimplementedUserServer
 }
 
+type clientInterface interface {
```

Comment:
maybe just `grafanaClient` like in other places?

Reply:
Done